roles/default/kiali-deploy/templates/kubernetes/role.yaml

```diff
@@ -45,6 +47,7 @@ rules:
   - get
   - list
   - watch
+  - patch
```

Comment:
Need to add "patch" here to the operator role, line 218: https://github.com/kiali/kiali-operator/blob/v1.21/deploy/role.yaml#L218